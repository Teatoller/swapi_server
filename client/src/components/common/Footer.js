import React from "react";

const Footer = () => {
  return (
    <footer
      id="footer"
      className="mx-auto text-center text-warning border-top border-warning"
    >
      <span> {new Date().getFullYear()}&copy;Swapi</span>
    </footer>
  );
};

export default Footer;
