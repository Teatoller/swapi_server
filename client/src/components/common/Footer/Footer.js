import React from "react";

const Footer = () => (
    <footer
      id="footer"
      className="mx-auto text-center text-warning border-top border-warning"
    >
      <span> {new Date().getFullYear()}&copy;Swapi</span>
    </footer>
  );

export default Footer;
