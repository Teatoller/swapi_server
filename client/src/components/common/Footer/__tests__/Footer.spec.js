import React from "react";
import { render, screen } from '@testing-library/react';
import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("Footer", () => {
  test('renders Footer component', () => {
    render(<Footer />);
    expect(screen.getByRole('contentinfo')).toBeInTheDocument();
  });

  it("snapshot renders correctly", () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
