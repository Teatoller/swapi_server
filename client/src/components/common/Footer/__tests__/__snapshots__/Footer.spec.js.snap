// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot renders correctly 1`] = `
<footer
  className="mx-auto text-center text-warning border-top border-warning"
  id="footer"
>
  <span>
     
    2021
    Â©Swapi
  </span>
</footer>
`;
